src/library/dlgtrackinfomulti.cpp

```diff
@@ -17,24 +17,24 @@
 #include "util/datetime.h"
 #include "util/duration.h"
 #include "util/optional.h"
+#include "util/stringformat.h"
 #include "widget/wcoverartlabel.h"
 #include "widget/wcoverartmenu.h"
 #include "widget/wstarrating.h"
 
 namespace {
 
-const QString kVariousText = char('<') + QObject::tr("various") + char('>');
+const QString kVariousText = QChar('<') + QObject::tr("various") + QChar('>');
 const char* kOrigValProp = "origVal";
 const QString kClearItem = QStringLiteral("clearItem");
 
 /// If value differs from the current value, add it to the list.
 /// If new and current are identical, keep only one. Later on we can use the
 /// item count to maybe join the list and format the font accordingly.
-/// List can be a QStringList or a QList<double>
 template<typename T>
-void appendToTagList(QList<T>* pList, const T& value) {
+void addToTagSet(QSet<T>* pList, const T& value) {
     if (pList->isEmpty() || !pList->contains(value)) {
-        pList->append(value);
+        pList->insert(value);
     }
 }
 
```

Comment:
Now that we are using QSet, you don't need this function at all ("inserts **if value isn't already in the set**, and returns an iterator pointing at the inserted item.") https://doc.qt.io/qt-6/qset.html#insert

Reply:
thanks, wasn't aware of that. just read that QSet may contain duplicates.